doc/bash/npm_aliases.md

```diff
@@ -0,0 +1,21 @@
+# Bash aliases for npm #
+
+-[Npm](#npm)
```

Comment:
Please add a space after each dash. See how the page looks right now: https://github.com/PaulBuda/dotaliases/blob/npm-aliases/doc/bash/npm_aliases.md